client/post-editor/editor-drawer/index.jsx

```diff
@@ -35,17 +34,46 @@ import stats from 'lib/posts/stats';
 import siteUtils from 'lib/site/utils';
 import { setExcerpt } from 'state/ui/editor/post/actions';
 import QueryPostTypes from 'components/data/query-post-types';
-import { getSelectedSite } from 'state/ui/selectors';
-import { getPostTypes } from 'state/post-types/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import { getPostType } from 'state/post-types/selectors';
 import config from 'config';
 import EditorDrawerFeaturedImage from './featured-image';
 import EditorDrawerTaxonomies from './taxonomies';
 
+/**
+ * Constants
+ */
+
+/**
+ * A mapping of post type to hard-coded post types support. These values are
+ * used as fallbacks if the REST API type entity has not been retrieved, and
+ * prevent the post type query component from being rendered.
+ *
+ * @type {Object}
+ */
```

Comment:
>Maybe add what version/which endpoint this is coming from in case we need to update the fallback in the future.
 

Added link to endpoint documentation in rebased 95f6d224bd3a2a5e0687c6806dc38c9f955197ed.